src/FileWatcher/index.js

```diff
@@ -77,6 +84,8 @@ class FileWatcher extends EventEmitter {
   }
 
   _createWatcher(rootConfig) {
+    var _this = this;
```

Comment:
what about `const forward = (...args) => this.emit('all', ...args)` and use that further down?